packages/injectable/src/dependency-injection-container/createContainer.registration.test.js

```diff
@@ -16,22 +16,67 @@ describe('createContainer.registration', () => {
     expect(actual).toBe('some-injected-instance');
   });
 
-  it('throws an error with the key if the value behind has no default export', () => {
-    const context = new Map();
+  it('given injectable file with no default export, when auto-registering, throws with name of faulty file', () => {
+    const requireContextStub = Object.assign(
+      () => ({
+        notDefault: 'irrelevant',
+      }),
+      {
+        keys: () => ['./some.injectable.js'],
+      },
+    );
+
+    expect(() => createContainer(() => requireContextStub)).toThrowError(
+      /\.\/some\.injectable\.js/,
+    );
+  });
 
-    context.set("./foobar.injectable.ts", {
-      notDefault: getInjectable({
-        id: 'irrelevant',
-        instantiate: () => 'some-injected-instance',
+  it('given injectable file with default export without id, when auto-registering, throws with name of faulty file', () => {
+    const requireContextStub = Object.assign(
+      () => ({
+        default: 'irrelevant',
       }),
-    });
+      {
+        keys: () => ['./some.injectable.js'],
+      },
+    );
+
+    expect(() => createContainer(() => requireContextStub)).toThrowError(
+      /\.\/some\.injectable\.js/,
+    );
+  });
 
-    expect(() => createContainer(() => Object.assign(
-      (key) => context.get(key),
+  it('given injectable file with default export with in but without instantiate, when auto-registering, throws with name of faulty file', () => {
+    const requireContextStub = Object.assign(
+      () => ({
+        default: {
+          id: 'foobar',
+        },
+      }),
       {
-        keys: () => context.keys(),
-      }
-    ))).toThrowError(/\.\/foobar\.injectable\.ts/);
+        keys: () => ['./some.injectable.js'],
+      },
+    );
+
+    expect(() => createContainer(() => requireContextStub)).toThrowError(
+      /\.\/some\.injectable\.js/,
+    );
+  });
+
+  it('given injectable file with default export of correct shape, when auto-registering, does not throw', () => {
+    const requireContextStub = Object.assign(
+      () => ({
+        default: {
+          id: 'foobar',
```

Comment:
Note: where value is relevant, instead of 'foobar', one has to opportunity to use a specific name to improve descriptions of failing tests, and thus ease fixing them. Should the value here be relevant, this codebase's convention would call it "some-injectable-id".